doc/rst/build.rst

```diff
@@ -142,6 +142,16 @@ system path then specifying the CART and DAOS paths is unnecessary.
      -DENABLE_DAOS=ON
    make install
 
+To build with HDF5 support, and to build tools requiring HDF5 support,
+first make sure HDF5 1.2+ and vol-daos is installed.
+Then, add the following flag(s) during CMake. If HDF5 is installed under a standard
```

Comment:
@daltonbohning I think we should state that to run the serialization requires only HDF5 (1.2+), and then if you want to copy HDF5 containers it requires HDF5 1.8+ (I think or 1.7?) and the daos-vol. Since the VOL is not required for serialization.

Reply:
@dsikich Updated. I went back through our conversation, and it's version 1.8 that is required.